src/app/(private)/map/[id]/components/PrivateMapNavbar.tsx

```diff
@@ -169,84 +169,86 @@ export default function PrivateMapNavbar() {
   };
 
   return (
-    <nav className="flex justify-between items-center p-4 z-10 h-14 border-b border-neutral-200 bg-white">
-      <div className="flex items-center gap-4">
-        <Link href="/dashboard" className="flex items-center gap-2">
-          <Image src="/logo.svg" alt="Mapped" width={24} height={24} />
-        </Link>
-        <div className="flex items-center gap-2 text-sm text-neutral-600">
-          <Link
-            href="/dashboard"
-            className="text-neutral-500 hover:text-neutral-800"
-          >
-            Maps
-          </Link>
-          <ChevronRight className="w-4 h-4 text-neutral-400" />
-          <div className="flex items-center gap-1">
-            {isEditingName ? (
-              <>
-                <input
-                  type="text"
-                  value={mapName || ""}
-                  className="px-3 py-1 border border-neutral-300 rounded text-sm"
-                  onChange={(e) => setMapName(e.target.value)}
-                />
-                <Button variant="outline" size="sm" onClick={onSubmitSaveName}>
-                  Save
-                </Button>
-                <Button
-                  variant="outline"
-                  size="sm"
-                  onClick={() => setIsEditingName(false)}
-                >
-                  Cancel
-                </Button>
-              </>
-            ) : (
-              <p className="truncate max-w-[300px]">
-                {mapName || "Loading..."}
-              </p>
-            )}
+    <Navbar>
```

Comment:
I created a reusable Navbar and had to change some flex containers below because of that, there's no other changes here!